### DISCUSSION TITLE
adding a squash option to kaniko build

### DISCUSSION THREAD

**Discussion by danilo-patrucco** [GoogleContainerTools/kaniko#3185] [General]
Hello,

Is there any interest in adding a squash option to kaniko ?
We have the need to mask off the layers to avoid issues with scanning tools like trivy, and squashing would help since copying the filesystem between containers in a multistage build (to remove all the layers in the history) does not work.

Thank you very much

**Comment by rofr** [comment]
https://github.com/GoogleContainerTools/kaniko?tab=readme-ov-file#flag---single-snapshot
